docs/implementers/enable-sso.md

```diff
@@ -1 +1,111 @@
+
+!!! note
+    The `ozone-realm.json` file configures the Keycloak (IdP) by defining all client applications that will use Keycloak for authentication. Make sure each application (OpenMRS, Odoo, SENAITE) has a corresponding client entry in this file, including the correct redirect URIs and required settings.
+
+### OpenMRS (EMR)
+
+OpenMRS integrates with Keycloak for SSO using the [OpenMRS OAuth2 Login Module](https://github.com/openmrs/openmrs-module-oauth2login#openmrs-oauth-20-login-module). This module is included by default but is initially disabled, you can enable it by setting the `oauth2.enabled` property to true in the `oauth2.properties` file.
+
+!!! note
+     Enabling SSO in OpenMRS effectively disables basic authentication, so users will authenticate through the Keycloak login page rather than using traditional username and password prompts.
+
+#### Configurations
+
+- `oauth2.properties` - Located at `<your-distribution>/distro/configs/openmrs/properties/`. This file contains Keycloak
+  server details, client ID, client secret, and other OAuth2 settings.
+- Environment Variables - Set via Docker Compose environment variables in the `docker-compose-openmrs-sso.yml` file.
+- `ozone-frontend-sso.json` - Located at `<your-distribution>/distro/configs/openmrs/frontend_config/`. This file
+  contains frontend configurations for SSO, which configures `esm-login-app` frontend module to use OAuth2 login
+  provider as well as login and logout URLs.
+- `ozone-realm.json` - Located at `<your-distribution>/distro/configs/openmrs/keycloak/`. Ensure that the OpenMRS client
+  is defined in this file with the correct redirect URIs.
+- `oauth2-login-props.xml` - Located at `distro/configs/openmrs/initializer_config/globalproperties/`. This file
+  contains global properties for the OAuth2 login module. Specifically, it contains the
+  `oauth2login.redirectUriAfterLogin` property, which is used to redirect users to the correct URL after login.
+
+### Odoo (ERP)
+
+Odoo integrates with Keycloak using [`auth_oauth`](https://github.com/OCA/server-auth/tree/18.0/auth_oidc) addon by OCA community to provide SSO support. However, you can enable it by adding the `auth_oauth` addon to the `ODOO_ADDONS`environment variable.
+
+#### Configurations:
+
+- `auth.oauth.provider.csv` - Located at `<your-distribution>/configs/odoo/initializer_config/auth_providers/`. This
+  file is used to configure the OAuth2 provider in Odoo. It contains the Keycloak server details, client ID, client
+  secret, and other OAuth2 settings.
+- `ODOO_ADDONS` - Set via Docker Compose environment variables in the `docker-compose-odoo-sso.yml` file to include the
+  `auth_oauth` addon. Alternatively, this can be done by setting the `ODOO_ADDONS` environment variable in the `.env`
+  file.
+- `ozone-realm.json` - Located at `<your-distribution>/distro/configs/odoo/keycloak/`. Ensure that the Odoo client is
+  defined in this file with the correct configs and redirect URIs.
```

Comment:
```suggestion
- `ODOO_ADDONS` - Set via Docker Compose environment variables in the `docker-compose-odoo-sso.yml` file to include the `auth_oauth` addon. Alternatively, this can be done by setting the `ODOO_ADDONS` environment variable in the `.env` file.
- `ozone-realm.json` - Located at `<your-distribution>/distro/configs/odoo/keycloak/`. Ensure that the Odoo client is defined in this file with the correct configs and redirect URIs.
```